tee-worker/ts-tests/integration-tests/common/utils/crypto.ts

```diff
@@ -145,7 +145,11 @@ export class BitcoinSigner implements Signer {
         return new Promise((resolve, reject) => {
             if (isString(message)) {
                 const sig = new bitcore.Message(message).sign(this.keypair);
-                resolve(bufferToU8a(Buffer.from(sig, 'base64')));
+                let buf = Buffer.from(sig, 'base64');
+                if (buf.length === 64) {
```

Comment:
Is it possible to have more idiomatic code by using e.g. a fixed length buffer? Something like `[u8; 65]` in rust to make sure it's always 65 bytes 🤔

Reply:
>I haven't checked the lib impl and am not sure if the k is random or not
I checked:
<img width="855" alt="image" src="https://github.com/litentry/litentry-parachain/assets/104152026/ce219665-d561-4684-8132-8a8686d6b02a">

Reply:
After testing, it is not just the error case where `buffer.length === 64`. In fact, there will also be cases where it equals 63 (not excluding other cases, 63 is the result of 1000 loops). Therefore, change to using do while to ensure getting u8aArray 65 sig.

```
1) Test Identity (bitcoin direct invocation)
       check idGraph from sidechain storage before linking:
     Error: createType(LitentryMultiSignature):: Enum(bitcoin):: Expected input with 65 bytes (520 bits), found 63 bytes
```

The way of creating a buffer seems to be impractical, as it will change the content of the signature, such as this way:
```
let buf = Buffer.from(sig, 'base64');
Buffer.alloc(65)
buf.copy(resultBuf, 65 - buf.length);
```
Will get an invalid error: 
`Got response: {"value":"TGetterIsNotAuthorized","do_watch":false,"status":"Error"}`

So my initial approach was also incorrect. The reason why the test passed is that when I [console.log here](https://github.com/litentry/litentry-parachain/pull/2553/commits/d87def6fd3655f4f555c64fc50db3091e7690c9c#diff-cfd728a08e2960d755d2a6976c5b9a2863031978a63a2390673ec7d9305da3c4L126-L127), actually called the sign method. Due to the uncertainty of the signature result, after two console calls in front, the third call [here](https://github.com/litentry/litentry-parachain/blob/dev/tee-worker/ts-tests/integration-tests/common/di-utils.ts#L126) happened to match the sig we used, so the test passed strangely.